tools/tests/merger/fixtures/spec/namespaces/shelter.yaml

```diff
@@ -24,22 +24,22 @@ paths:
         '200':
           $ref: '#/components/responses/adopt@200'
 components:
-    requestBodies:
-      adopt: {}
-    parameters:
-      adopt::path.animal:
-        name: animal
-        in: path
+  requestBodies:
+    adopt: {}
+  parameters:
+    adopt::path.animal:
+      name: animal
+      in: path
+      schema:
+        $ref: '../schemas/animals.yaml#/components/schemas/Animal'
+    adopt::path.docket:
+      name: docket
+      in: path
+      schema:
+        type: number
+  responses:
+    adopt@200:
+      description: ''
+      application/json:
         schema:
-          $ref: '../schemas/animals.yaml#/components/schemas/Animal'
-      adopt::path.docket:
```

Comment:
We discussed in this [PR](https://github.com/opensearch-project/opensearch-api-specification/pull/271/files) about allowing (or even enforcing) an empty line between properties to help discern large and complex properties. Is this still the case?

Reply:
Undid.